src/examples/java/io/nats/examples/ExampleUtils.java

```diff
@@ -59,14 +59,19 @@ public void slowConsumerDetected(Connection conn, Consumer consumer) {
 
     public static void createTestStream(JetStream js, String streamName, String subject)
             throws TimeoutException, InterruptedException {
+        createTestStream(js, streamName, subject, StorageType.File);
+    }
+
+    public static void createTestStream(JetStream js, String streamName, String subject, StorageType storageType)
+            throws TimeoutException, InterruptedException {
 
         // Create a stream, here will use a file storage type, and one subject,
         // the passed subject.
-        StreamConfiguration sc = StreamConfiguration.builder().
-            name(streamName).
-            storageType(StorageType.File).
-            subjects(new String[] { subject }).
-            build();
+        StreamConfiguration sc = StreamConfiguration.builder()
+                .name(streamName)
+                .storageType(storageType)
+                .subjects(subject)
+                .build();
```

Comment:
convention is to put the dot at the beginning. also when concatenating strings with + it goes at the beginning, or using && for multi line if. It makes it easier to comment out a line too.